ansible/molecule/ec2_gpu/molecule.yml

```diff
@@ -0,0 +1,69 @@
+---
+scenario:
+  name: ec2_gpu
+driver:
+  name: ec2
+platforms:
+  - name: konvoyimage-centos7.9-${USER:-ci}-${HOSTNAME:-local}
+    image_search_name: "CentOS 7.9.2009 x86_64"
+    image_search_owner: "125523088429"
+    region: us-east-1
+    instance_type: p2.xlarge
```

Comment:
Do we need to execute builds on `p2.xlarge` instance types?

Reply:
yup with that the tests ensures the kernel successfully loaded the module as nvidia-smi is being used